### PR TITLE
[Zone] Add LavaDamage and MinLavaDamage support to ZoneHeader

### PR DESCRIPTION
**Pull request by Cole-SoD**
`
/*Add lava damage variables to zone table*/

ALTER TABLE `zone` ADD `lava_damage` INT(11) NULL DEFAULT '50' AFTER `underworld_teleport_index`, ADD `min_lava_damage` INT(11) NOT NULL DEFAULT '10' AFTER `lava_damage`;
`

### CODE REVIEW

File: utils/sql/db_update_manifest.txt [hunk 1]
```diff
@@ -426,6 +426,7 @@
 9170|2021_03_03_instance_safereturns.sql|SHOW TABLES LIKE 'character_instance_safereturns'|empty|
 9171|2021_03_30_remove_dz_is_current_member.sql|SHOW COLUMNS FROM `dynamic_zone_members` LIKE 'is_current_member'|not_empty|
 9172|2021_05_21_shared_tasks.sql|SHOW TABLES LIKE 'shared_tasks'|empty|
+9173|2021_09_12_zone_lava_damage.sql|SHOW COLUMNS from `zone` WHERE field = 'lava_damage' or field = 'min_lava_damage'|empty|
```
Comment (Akkadius) [inline utils/sql/db_update_manifest.txt:429]: We only need to check for one column here

**Review by Akkadius — APPROVED** [review]

**Comment by Akkadius** [comment]
Minor comment. Was this tested?

**Comment by Cole-SoD** [comment]
> Minor comment. Was this tested?
> 
> Otherwise looks good! Thanks for putting in the effort to put this up

This was not tested. 

> Minor comment. Was this tested?
> 
> Otherwise looks good! Thanks for putting in the effort to put this up

Not tested. I installed EQEmu locally to test just today. The auto updating of the DB is not working, few errors there

**Comment by Cole-SoD** [comment]
> Minor comment. Was this tested?
> 
> Otherwise looks good! Thanks for putting in the effort to put this up

image

Tested locally. I assume you run the DB queries, and when someone's local binaries are updated they automatically pull the database changes using eqemu_server.pl.

**Comment by Akkadius** [comment]
> > Minor comment. Was this tested?
> > Otherwise looks good! Thanks for putting in the effort to put this up
> 
> Not tested. I installed EQEmu locally to test just today. The auto updating of the DB is not working, few errors there. (I am just assuming that the binary makes changes to the database structure?)

Auto update won't work until things are merged. The manifest gets pulled from master 

`eqemu_server.pl` is what pulls down the manifest and conditionally updates your database based on the compiled binary database version that comes back from the `world` binary

**Comment by Akkadius** [comment]
> > Minor comment. Was this tested?
> > Otherwise looks good! Thanks for putting in the effort to put this up
> 
> image
> 
> Tested locally, this is skyfire with 100 min_lava_damage and 500 lava_damage. (Original damage was 45 with default values) I assume you run the DB queries, and when someone's local binaries are updated that will automatically pull the database changes using eqemu_server.pl.

Correct and the manifest is in master

File: utils/sql/db_update_manifest.txt [hunk 2]
```diff
@@ -426,6 +426,7 @@
 9170|2021_03_03_instance_safereturns.sql|SHOW TABLES LIKE 'character_instance_safereturns'|empty|
 9171|2021_03_30_remove_dz_is_current_member.sql|SHOW COLUMNS FROM `dynamic_zone_members` LIKE 'is_current_member'|not_empty|
 9172|2021_05_21_shared_tasks.sql|SHOW TABLES LIKE 'shared_tasks'|empty|
+9173|2021_09_12_zone_lava_damage.sql|SHOW TABLES LIKE 'lava_damage'|empty|
```
Comment (Akkadius) [inline utils/sql/db_update_manifest.txt:429]: This is not going to work

Try something like this 

```sql
SHOW COLUMNS FROM `zone` LIKE 'lava_damage'
```

Also your filename does not match what is committed 

Manifest `2021_09_12_zone_lava_damage.sql` vs File `2021_09_14_zone_lava_damage.sql`

**Review by Akkadius — CHANGES_REQUESTED** [review]
Left a few comments

**Review by Kinglykrab — APPROVED** [review]

**Review by mackal — APPROVED** [review]

**Review by Akkadius — APPROVED** [review]

**Pull request merged**